src/components/ReduxAppWrapper/store/reducers/globals.js

```diff
@@ -57,6 +58,8 @@ export default function globals(state = initialState, action) {
           ...action.payload
         }
       }
+    case constants.SET_ONFIDO_LOGO_DISABELD:
```

Comment:
I personally prefer to use a positive naming for a boolean property, so in this case I would use `SET_ONFIDO_LOGO_ENABLED` which would be `true` by default and it would return `{...state, hideOnfidoLogo: !action.payload}`

Reply:
I changed this to `HIDE_ONFIDO_LOGO` as Kimmi suggested as it seems even more clear